app/controllers/concerns/foreman_teamdynamix/hosts_controller_extensions.rb

```diff
@@ -0,0 +1,21 @@
+module ForemanTeamdynamix
+  # Example: Plugin's HostsController inherits from Foreman's HostsController
+  module HostsControllerExtensions
+    extend ActiveSupport::Concern
+
+    def team_dynamix
+      find_resource
+      render partial: 'foreman_teamdynamix/hosts/team_dynamix', :locals => { :host => @host }
+    end
```

Comment:
Why aren't we handling the ajax errors?
https://github.com/MiamiOH/foreman_custom_tab/blob/master/app/controllers/concerns/foreman_custom_tab/hosts_controller_extensions.rb#L12